### ISSUE TITLE
TASK [copy kubectl binary to ansible host]

### ISSUE THREAD

**Issue by ZelChief** [NVIDIA/deepops#725]
```bash
TASK [copy kubectl binary to ansible host] ***********
fatal: [mgmt01]: FAILED! => changed=false 
  msg: 'Failed to find required executable rsync in paths: /usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/sbin:/usr/sbin:/usr/local/sbin'
```

![Снимок экрана 2020-10-20 в 22 31 37](https://user-images.githubusercontent.com/5526126/96635203-3b535700-1324-11eb-89ab-5e4bbb201d3b.png)

But 

```bash
tpadmin@mgmt01:~$ sudo apt-get install -y rsync
Reading package lists... Done
Building dependency tree       
Reading state information... Done
rsync is already the newest version (3.1.2-2.1ubuntu1.1).
0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.
```

**Comment by ajdecon** [comment]
It looks like this play is attempting to run `rsync` on the Ansible control host, i.e. the machine where you are running `ansible-playbook`. If this is not the same as `mgmt01` (the Kubernetes management node), do you have rsync installed on the control host?

**Comment by ZelChief** [comment]
OK, working 👍 
![Снимок экрана 2020-10-21 в 0 03 26](https://user-images.githubusercontent.com/5526126/96643926-f386fc80-1330-11eb-84d1-27f2ae4be337.png)

I'm new in DGX-2 system. @ajdecon, how to get in Kubernetes dashboard?  http://ip_of_mgmt01:8001 in empty :)

**Comment by ajdecon** [comment]
The Kubernetes dashboard appears to be exposed on NodePort 31443. You can check the services exposed for k8s admin like this:

```
$ kubectl -n kube-system get services
NAME                                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                        AGE
coredns                                         ClusterIP   10.233.0.3      <none>        53/UDP,53/TCP,9153/TCP         10m
dashboard-metrics-scraper                       ClusterIP   10.233.8.118    <none>        8000/TCP                       9m55s
kube-prometheus-stack-coredns                   ClusterIP   None            <none>        9153/TCP                       3m27s
kube-prometheus-stack-kube-controller-manager   ClusterIP   None            <none>        10252/TCP                      3m27s
kube-prometheus-stack-kube-etcd                 ClusterIP   None            <none>        2379/TCP                       3m27s
kube-prometheus-stack-kube-proxy                ClusterIP   None            <none>        10249/TCP                      3m27s
kube-prometheus-stack-kube-scheduler            ClusterIP   None            <none>        10251/TCP                      3m27s
kube-prometheus-stack-kubelet                   ClusterIP   None            <none>        10250/TCP,10255/TCP,4194/TCP   2m45s
kubernetes-dashboard                            NodePort    10.233.50.198   <none>        443:31443/TCP                  9m55s
```

And you can get a token for accessing the dashboard using this script:

```
$ ./scripts/k8s/deploy_dashboard_user.sh
service/kubernetes-dashboard patched (no change)

Dashboard is available at: https://10.0.0.2:31443

Access token: <redacted>
```

**Comment by ZelChief** [comment]
Great! Thanks for your work 👍
